src/pages/projects.js

```diff
@@ -1,11 +1,32 @@
+import { PageHeader } from '../components/page-header/page-header'
+import { useState } from 'react'
 import { ProjectContent } from '../components/project-content/project-content'
 export const ProjectsPage = () => {
+    const [datas, setDatas] = useState([
+        {
```

Comment:
This array is a nice solution👌it would be better if it's extracted to a separate file.

For instance there can be projects-data.js as follows

```javascript
export projects =[{},{},{}]; //same object in useState

```

Then this data can be used as follows

```javascript
import {projects} from "./projects-data"

```